app/assets/templates/home.html

```diff
@@ -1,5 +1,8 @@
-<div><header></header></div>
+<header></header>
```

Comment:
Yes it is a directive
bad name I know. I dont want to touch it. Dale was changing the naming of the folders and it went from top-Header to header...